src/services/db/GitFileSystemService.ts

```diff
@@ -82,26 +86,22 @@ export default class GitFileSystemService {
 
   // Determine if the folder is a valid Git repository
   isValidGitRepo(repoName: string): ResultAsync<boolean, GitFileSystemError> {
-    const safeExistsSync = Result.fromThrowable(fs.existsSync, (error) => {
-      logger.error(`Error when checking if ${repoName} exists: ${error}`)
-
-      if (error instanceof GitError || error instanceof Error) {
-        return new GitFileSystemError("Unable to determine if directory exists")
-      }
-
-      return new GitFileSystemError("An unknown error occurred")
-    })
-
-    return safeExistsSync(`${EFS_VOL_PATH}/${repoName}`)
-      .andThen((isFolderExisting) => {
-        if (!isFolderExisting) {
```

Comment:
why are these removed?

Reply:
We've opted to use `gitFilePathStats` instead of `fs.existsSync`, so these have been removed!